src/net.h

```diff
@@ -769,11 +769,30 @@ class CNode
 class NetEventsInterface
 {
 public:
-    virtual bool ProcessMessages(CNode* pnode, std::atomic<bool>& interrupt) = 0;
-    virtual bool SendMessages(CNode* pnode) EXCLUSIVE_LOCKS_REQUIRED(pnode->cs_sendProcessing) = 0;
+    /** Initialize a peer (setup state, queue any initial messages) */
```

Comment:
Maybe precise that's network-related state, and that we queue ready-to-send messages, not received messages yet.